test/torch/mpc/falcon/test_falcon.py

```diff
@@ -26,3 +28,98 @@ def test_conv2d_private(workers):
         falcon.conv2d(x, y, padding=1).reconstruct()
         == torch.tensor([[[[123, 180, 132], [224, 312, 224], [132, 180, 123]]]])
     ).all()
+
+
+def test_private_compare(workers):
+    l = 2 ** 5
+    L = 2 ** l
+    p = 37
+
+    players = (workers["alice"], workers["bob"], workers["james"])
+    x_bit_sh = (
+        FalconHelper.decompose(torch.LongTensor([13]), L).share(*players, protocol="falcon").child
+    )
+    r_bit = (
```

Comment:
`r` is public in the protocol, no need to secret share it

Reply:
Yeah that's fair.